lib/connect-redis.js

```diff
@@ -62,11 +90,16 @@ module.exports = function(session) {
       if (this.disableTouch) return cb()
 
       let key = this.prefix + sid
-      this.client.expire(key, this._getTTL(sess), (err, ret) => {
-        if (err) return cb(err)
-        if (ret === 1) return cb(null, 'OK')
-        this.set(sid, sess, cb) // fallback to setex if the key has expired
-      })
+      let expires = (sess && sess.cookie && sess.cookie.expires) || ''
+      if (this.client.eval) {
```

Comment:
note: this ensure that the redis-client support eval, else we fallback to the old behavior